modules/calib3d/include/opencv2/calib3d.hpp

```diff
@@ -517,6 +517,7 @@ enum { CALIB_NINTRINSIC          = 18,
        CALIB_FIX_TAUX_TAUY       = 0x80000,
        CALIB_USE_QR              = 0x100000, //!< use QR instead of SVD decomposition for solving. Faster but potentially less precise
        CALIB_FIX_TANGENT_DIST    = 0x200000,
+       CALIB_FIX_EXTRINSIC       = (1 << 23),
```

Comment:
There is `CALIB_RECOMPUTE_EXTRINSIC` for fisheye. Also need to align with new flags in 5.x.